packages/core/src/browser/xhrProxy.spec.ts

```diff
@@ -209,4 +209,26 @@ describe('xhr proxy', () => {
       },
     })
   })
+
+  it('should track only the second request (the first request is canceled)', (done) => {
```

Comment:
FMU, there was a bit more to it.
The first one is cancel only if the second one start before its end.
Should not we reflect that in the test and test both cases? or only rely on e2e scenario?

Reply:
You'r right, because this behaviours are tied to the browser, I'd be in favour of relying only on the e2e tests. 
In my opinion, it's the only reliable way to know if our implementation comply with browsers behaviours.

Reply:
let's remove this one then